packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -117,19 +113,6 @@ export class VaMemorableDate {
   @Prop({ mutable: true }) invalidMonth: boolean = false;
   @Prop({ mutable: true }) invalidYear: boolean = false;
 
-  private focusInvalidFields = () => {
```

Comment:
I added this as part of [this](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1593) card, but the individual fields cannot be focussed when invalid because it creates the keyboard trap. This behavior is also consistent with other form fields, focus is not sent back to the field when it is invalid.